spec/views/conferences/new.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "conferences/new", type: :view do
+  before(:each) do
+    assign(:conference, Conference.new(
+      :name => "MyString"
```

Comment:
Indent the first parameter one step more than `Conference.new(`.<br>Use the new Ruby 1.9 hash syntax.